packages/lib/src/components/Dropin/Dropin.tsx

```diff
@@ -139,6 +140,14 @@ class DropinElement extends UIElement<DropinElementProps> {
         return null;
     }
 
+    /**
+     * handleOrder is implemented so we don't trigger a callback like in the components
+     * @param response - PaymentResponse
+     */
+    protected handleOrder = (response: PaymentResponse): void => {
```

Comment:
I think order could be destructured from response since nothing from response is being used.